pkgs/by-name/ap/aphorme/package.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, fontconfig
+, pkg-config
+, wayland
+, libxkbcommon
+, libGL
+, xorg
+, testers
+, aphorme
+, autoPatchelfHook
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "aphorme";
+  version = "0.1.19";
+
+  src = fetchFromGitHub {
+    owner = "Iaphetes";
+    repo = "aphorme_launcher";
+    rev = "v${version}";
```

Comment:
Thanks!
I learned something again :)
Will change in amended commit to keep commit-count down.

Reply:
It's because the default `callPackage` passes in everything that's under the top-level as well as under `xorg`:

https://github.com/NixOS/nixpkgs/blob/6c0a5bef965318da037febf4114e4dfe52ada460/pkgs/top-level/splice.nix#L137-L140